<?xml version="1.0" ?>

<launch>

  <!-- Run this node -->
  <node pkg="hockey_shot" type="hockey_shot" name="hockey_shot" output="screen"/>

  <!-- Rviz config -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find hockey_shot)/rviz/hockey_shot.rviz"/> 

</launch>
